rpc/src/rpc.rs

```diff
@@ -510,6 +510,62 @@ impl JsonRpcRequestProcessor {
         })
     }
 
+    pub fn get_secondary_index_key_size (
+        &self,
+        index_key: &Pubkey,
+        commitment: Option<CommitmentConfig>
+    ) -> Result<RpcResponse<Option<HashMap<JsonAccountIndex, usize>>>>
+    {        
+        // Grab RPC server's secondary index config
+        // Exit if secondary indexes are not enabled
+        let secondary_index_config = 
+            if !self.config.account_indexes.is_empty() {
+                &self.config.account_indexes
+            }
+            else { return None.ok_or(Error::from(RpcCustomError::SecondaryIndexDoesNotExist)); };
```

Comment:
I think an empty HashMap would be okay for this case as well. Callers of this method will be RPC operators, who should be intimately familiar with their secondary-index settings.